bitnami/airflow/templates/web/service.yaml

```diff
@@ -6,7 +6,7 @@ SPDX-License-Identifier: APACHE-2.0
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ include "common.names.fullname" . }}
+  name: {{ include "airflow.web.fullname" . }}
```

Comment:
Any special reason to change this service name?

Reply:
We already had the helper available (basically it is `"<common.names.fullname>-web"`), and it is used to populate the `execution_api_server_url`. I think it makes it easier to understand it is pointing to the `airflow-web` pod via its k8s service.